src/components/Collection/GridCollection.tsx

```diff
@@ -1,12 +1,15 @@
-import React from 'react';
+import React from "react";
 import ReactSVG from "react-svg";
 import gridIcon from "../../images/baseline-dashboard.svg";
 
-export const GridCollection = () => {
-  return(
-    <div className="collection-wrapper__main collection-wrapper__grid">
-        <ReactSVG path={gridIcon} className="collection-wrapper__main-icon"/>
-        <span className="collection-wrapper__main-name">Grid</span>
+export const GridCollection = ({ handleViewChange }) => {
+  return (
+    <div
+      className="collection-wrapper__main collection-wrapper__grid"
+      onClick={() => handleViewChange("grid")}
```

Comment:
Should be capitalized for constants: "GRID",

And put in a Typescript enum: `enum GridViewTypes`

Reply:
Done